src/dev-mock-service/http-mock-service/types.ts

```diff
@@ -0,0 +1,6 @@
+export type HttpOverride = {
```

Comment:
I think users end up interacting with this type a bit? if so, consider documenting in JSDocs.

Reply:
This is a purely internal type, used to filter overrides based on the type